app/timetables/tests/test_utils.py

```diff
@@ -0,0 +1,13 @@
+from django.test import TestCase
+
+from app.timetables.utils import timestamp_seconds
+
+
+class UtilsTest(TestCase):
+    """Test utils module."""
+
+    def test_timestamp_seconds(self):
```

Comment:
Looking good so far but I have some concerns here.

This test basically checks that the function, when called `n` times, returns `n` unique timestamps but doesn't test the behavior of the `timestamp_seconds` given certain conditions.

The way this has been implemented would make it challenging to test as the function is **impure**

You might want to update the implementation of the `timestamp_seconds` function. Perhaps make it a pure function